### PR TITLE
Update customer-key-overview.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a line to the SPO and ODB requirements saying the accounts need to have the appropriate licenses to assign DEP.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3323

File: microsoft-365/compliance/customer-key-overview.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ A data encryption policy defines the encryption hierarchy to encrypt data using
...
 
-**SharePoint Online, OneDrive for Business, and Teams files** If you're using the multi-geo feature, you can create up to one DEP per geo for your organization. You can use different Customer Keys for each geo. If you're not using the multi-geo feature, you can only create one DEP per tenant. When you assign the DEP, encryption begins automatically but can take some time to complete. Refer to the details in [Set up Customer Key](customer-key-set-up.md).
+**SharePoint Online, OneDrive for Business, and Teams files** If you're using the multi-geo feature, you can create up to one DEP per geo for your organization. You can use different Customer Keys for each geo. If you're not using the multi-geo feature, you can only create one DEP per tenant. When you assign the DEP, encryption begins automatically but can take some time to complete.Each user must have appropriate licenses in order to assign a DEP. Refer to the details in [Set up Customer Key](customer-key-set-up.md).
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-overview.md]:
```suggestion
**SharePoint Online, OneDrive for Business, and Teams files** If you're using the multi-geo feature, you can create up to one DEP per geo for your organization. You can use different Customer Keys for each geo. If you're not using the multi-geo feature, you can only create one DEP per tenant. When you assign the DEP, encryption begins automatically but can take some time to complete. Each user must have appropriate licenses in order to assign a DEP. Refer to the details in [Set up Customer Key](customer-key-set-up.md).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks

**Comment by KCCross** [comment]
This doesn't read clearly. Licenses are discussed elsewhere. Are you modifying this in response to customer request? The way it reads now, it sounds like users assign DEPs.

**Comment by yogkumgit** [comment]
@msbemba

**Comment by msbemba** [comment]
@KCCross Yes, This was a response to the Customer's request. Since there was ambiguity on whether the admin or the users need the appropriate licenses.

Thanks
Sri

**Comment by KCCross** [comment]
Still not following. For SPO you assign a DEP at the site level, not per user. Please reword this.

**Comment by yogkumgit** [comment]
@msbemba

**Comment by msbemba** [comment]
@KCCross Thanks for the update. I'll close this PR and update the user.

**Pull request closed without merging**